bizdemo/easy_note/pkg/errno/errno.go

```diff
@@ -21,12 +21,16 @@ import (
 )
 
 const (
-	SuccessCode             = 0
-	ServiceErrCode          = 10001
-	ParamErrCode            = 10002
-	LoginErrCode            = 10003
-	UserNotExistErrCode     = 10004
-	UserAlreadyExistErrCode = 10005
+	SuccessCode                = 0
+	ServiceErrCode             = 10001
+	ParamErrCode               = 10002
+	LoginErrCode               = 10003
+	UserNotExistErrCode        = 10004
+	UserAlreadyExistErrCode    = 10005
+	AuthorizationFailedErrCode = 10006
+	NoRouteErrCode             = 10007
+	NoMethodErrCode            = 10008
+	ServerPanicErrCode         = 10009
```

Comment:
ServerPanicErrCode seem equal to ServiceErrCode

Reply:
i got it, thx